eva-accession-release-automation/publish_release_to_ftp/publish_release_files_to_ftp.py

```diff
@@ -111,48 +111,49 @@ def get_folder_path_for_species(release_folder_base, species_release_folder_name
-    See FTP layout linkage between by_assembly and by_species folders in the link below:
-    https://docs.google.com/presentation/d/1cishRa6P6beIBTP8l1SgJfz71vQcCm5XLmSA8Hmf8rw/edit#slide=id.g63fd5cd489_0_0
-    """
     assembly_accession = current_release_assembly_info["assembly_accession"]
     species_release_folder_name = current_release_assembly_info["release_folder_name"]
-    public_release_species_folder = os.path.join(release_properties.public_ftp_current_release_folder,
-                                                 by_species_folder_name, species_release_folder_name)
-    run_command_with_output("Creating symlink from species folder {0} to assembly folder {1}"
-                            .format(public_release_species_folder, public_release_assembly_folder),
-                            'bash -c "cd {1} && ln -sfT {0} {1}/{2}"'.format(
-                                os.path.relpath(public_release_assembly_folder, public_release_species_folder),
-                                public_release_species_folder, assembly_accession))
+    public_release_assembly_species_folder = os.path.join(public_release_assembly_folder, species_release_folder_name)
+    public_release_species_assembly_folder = os.path.join(release_properties.public_ftp_current_release_folder,
+                                                 by_species_folder_name, species_release_folder_name, assembly_accession)
+    run_command_with_output(f"""Creating symlink from assembly folder {public_release_assembly_species_folder} to 
+                            species folder {public_release_species_assembly_folder}""",
+                            'bash -c "cd {0} && ln -sfT {1} {2}"'.format(public_release_assembly_folder,
+                                os.path.relpath(public_release_species_assembly_folder, public_release_assembly_folder),
+                                public_release_assembly_species_folder))
 
 
 def recreate_public_release_assembly_folder(assembly_accession, public_release_assembly_folder):
-    run_command_with_output("Removing release folder if it exists for {0}...".format(assembly_accession),
-                            "rm -rf " + public_release_assembly_folder)
-    run_command_with_output("Creating release folder for {0}...".format(assembly_accession),
-                            "mkdir -p " + public_release_assembly_folder)
+    run_command_with_output(f"Removing release folder if it exists for {assembly_accession}...",
+                            f"rm -rf {public_release_assembly_folder}")
+    run_command_with_output(f"Creating release folder for {assembly_accession}...",
+                            f"mkdir -p {public_release_assembly_folder}")
 
 
-def copy_current_assembly_data_to_ftp(current_release_assembly_info, release_properties,
-                                      public_release_assembly_folder):
+def copy_current_assembly_data_to_ftp(current_release_assembly_info, release_properties, public_release_assembly_folder,
```

Comment:
Is `public_release_assembly_folder` actually used here?

Reply:
Again this was updates as part of the commit which I forgot to push. 
Pushed now.